wagtail/admin/views/home.py

```diff
@@ -224,9 +225,9 @@ def get_context_data(self, parent_context):
                     locked=True,
                     locked_by=request.user,
                 ),
-                "can_remove_locks": UserPagePermissionsProxy(
-                    request.user
-                ).can_remove_locks(),
+                "can_remove_locks": PagePermissionPolicy().user_has_any_permission(
```

Comment:
Shouldn't this be `user_has_permission` (or else `"unlock"` should be in a list/set)?